docs/api.rst

```diff
@@ -11655,6 +11658,9 @@ Handling user notes
    :resjson str location: The location inside the application the note was taken.
    :resjson int last_update_timestamp: The timestamp the note was last updated.
    :resjson bool is_pinned: Whether the note has been pinned by the user or not.
+   :resjson int entries_found: The number of entries found for the current filter. Ignores pagination.
```

Comment:
shouldn't you also edit the docs to add an appropriate error in the PUT verb if the notes would go over the limit?